pkg/virtctl/vm/restart.go

```diff
@@ -65,28 +65,29 @@ func (o *Command) restartRun(args []string, cmd *cobra.Command) error {
 		fmt.Printf("Dry Run execution\n")
 	}
 
-	if cmd.Flags().Changed(gracePeriodArg) && !forceRestart {
-		return fmt.Errorf("Can not set gracePeriod without --force=true")
-	}
-
-	if forceRestart {
-		if cmd.Flags().Changed(gracePeriodArg) {
-			err = virtClient.VirtualMachine(namespace).ForceRestart(context.Background(), vmiName, &v1.RestartOptions{GracePeriodSeconds: &gracePeriod, DryRun: dryRunOption})
-			if err != nil {
-				return fmt.Errorf("Error restarting VirtualMachine, %v", err)
-			}
-		} else if !cmd.Flags().Changed(gracePeriodArg) {
-			return fmt.Errorf("Can not force restart without gracePeriod")
+	if cmd.Flags().Changed(gracePeriodArg) {
```

Comment:
Same here, the complexity can be reduced by treating the input validation errors first, then do the actions.

Reply:
Done